build.gradle

```diff
@@ -777,15 +777,6 @@ task product(type: Zip) {
         exclude '.git*'
       }
     }
-    // copy AQP jar from maven repo if project not present
-    if (!hasAQPProject) {
-      copy {
-        from examplesProject.configurations.testRuntime.filter {
-          it.getName().contains('snappy-aqp')
-        }
-        into "${snappyProductDir}/jars"
-      }
-    }
```

Comment:
Removing this means that one cannot run the test suite without having aqp checkout (i.e. no one outside the org)? The idea was to fetch the non-oss jar from maven and let the tests run in any case.